content/blog/2020-08-19-train-a-sequence-model-with-poutyne/index.en.md

```diff
@@ -0,0 +1,608 @@
+---
+title: Train a Recurrent Neural Network (RNN) with Poutyne
+author: David Beauchemin et Marouane Yassine
+date: '2020-08-18'
+slug: machine learning
+type: post
+categories: ["python", "machine learning"]
+tags: []
+description: "Address tagging with a RNN"
```

Comment:
could be more specific for SEO if this is the meta description. 

Reply:
SEO?

Reply:
Search Engine Optimization. So that it is higher in Google's results.